packages/EasyErrorHandler/src/Bridge/Laravel/translations/en/exceptions.php

```diff
@@ -8,5 +8,6 @@
     'conflict' => 'Conflict.',
     'forbidden' => 'Forbidden.',
     'not_found' => 'Not found.',
+    'not_valid' => 'Validation failed.',
```

Comment:
This is the Laravel translations file 😄

Reply:
I've updated the default user message for the `packages/EasyErrorHandler/src/Exceptions/ValidationException.php` exception that's why I added it here :-)